socorro/external/postgresql/bugs.py

```diff
@@ -40,23 +40,21 @@ def get(self, **kwargs):
             sql_params.append(tuple(params.signatures))
 
             sql = """/* socorro.external.postgresql.bugs.Bugs.get */
-                SELECT ba.signature, bugs.id
-                FROM bugs
-                    JOIN bug_associations AS ba ON bugs.id = ba.bug_id
+                SELECT signature, bug_id as id
+                FROM bug_associations
                 WHERE EXISTS(
-                    SELECT 1 FROM bug_associations
-                    WHERE bug_associations.bug_id = bugs.id
+                    SELECT 1 FROM bug_associations as ba
+                    WHERE ba.bug_id = bug_associations.bug_id
```

Comment:
So that's what I initially thought, but the sub-query actually is needed, because, we want all associations for all bugs that have the given signatures. Say, for example, we have this data: 

bug id | signature
--------|-------------
1 | foo
1 | bar
2 | foo

If you ask for bugs related to ``foo``, the service should actually return all three results, including the second one, ``1 | bar``, even though that one doesn't have ``foo`` as a signature.